new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/WebLogicImageTool.java

```diff
@@ -124,4 +149,46 @@ private String buildList(List<String> list) {
     }
     return strList;
   }
+  
+  /**
+   * Add WDT installer to the WebLogic Image Tool cache
+   * @return true if the command succeeds 
+   */
+  public boolean addInstaller() throws FileNotFoundException {
+    try {
+      checkFile(WDT_ZIP_PATH);
+    } catch (FileNotFoundException fnfe) {
+      logger.warning("Failed to create an image due to Exception: " + fnfe.getMessage());
```

Comment:
Same.  If the file isn't found then this should fail the test.  I don't want to have to search the test logs for warnings to figure out why something later failed.